static/404.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Cold Crypto</title>
+    <title>Ice Wallet</title>
```

Comment:
Ice Wallet is not deployed to GitHub Pages so we can safely ignore it for the time being.

```suggestion
    <title>Cold Crypto</title>
```